src/features/customerView/createLease/createLeaseModal.module.scss

```diff
@@ -0,0 +1,21 @@
+@import 'containers';
+@import 'spacings';
+
+.createLeaseModal {
+  width: container(m);
+
+  .body {
+    padding: units(2) 0;
+  }
+
+  .select {
+    width: 100%;
+    margin-bottom: units(2);
+  }
+
+  .buttons {
+    display: flex;
+    justify-content: space-between;
+    // margin-top: units(1);
```

Comment:
This commented line should probably be removed

Reply:
Removed